conf/eslint.json

```diff
@@ -88,10 +88,10 @@
         "no-return-assign": "off",
         "no-script-url": "off",
         "no-self-assign": "error",
-        "no-self-compare": "off",
+        "no-self-compare": "error",
```

Comment:
We can only change those during major releases. So the next change will be during 3.0 release.